cob_hardware_config/cob4-2/config/battery_monitor.yaml

```diff
@@ -2,9 +2,9 @@ threshold_warning: 20 # % of battery level
 threshold_error: 10 # % of battery level
 threshold_critical: 5 # % of battery level
 
-enable_light: true
+enable_light: false
```

Comment:
sound and light is totally annoying because bms always says that the battery is empty. And we do not want to have it in the msh scenario.